src/Client/Core/DependencyInjection/DurableTaskClientExtensions.cs

```diff
@@ -30,7 +30,7 @@ public static class DurableTaskClientExtensions
     {
         Check.NotNull(client);
         PurgeInstancesFilter filter = new(createdFrom, createdTo, statuses);
-        return client.PurgeInstancesAsync(filter, cancellation);
+        return client.PurgeAllInstancesAsync(filter, cancellation);
```

Comment:
In the places you observed, did they use "All" even for methods that accepted filters?

Reply:
Yes they did: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/compute/Azure.ResourceManager.Compute/src/Generated/VirtualMachineCollection.cs#L231